03-functions-and-loops/invest.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def invest(initial_amount, interest_rate, duration):
+    amount = initial_amount
+
+    print("Principal amount: ", initial_amount)
+
+    print("Annual return rate: ", interest_rate)
+
+    for year in range(1, duration + 1):
+        compound_rate = amount * interest_rate
+        amount += compound_rate
+
+        print("Year ", year, ":", amount)
+
+    margin = amount - initial_amount
+
+    print("You made in {} years ${}".format(duration, margin))
+    print()
```

Comment:
If you want to add a separating new line between each `invest` call I suggest moving this call to the main function, as this call is only relevant if additional calls to `invest` are made, which is done in `main`.